tests/func/test_run_multistage.py

```diff
@@ -315,3 +318,43 @@ def test_run_without_cmd(kwargs):
     with pytest.raises(InvalidArgumentError) as exc:
         Repo().run(**kwargs)
     assert "command is not specified" == str(exc.value)
+
+
+def test_run_overwrite_order(tmp_dir, dvc, run_copy):
+    tmp_dir.gen({"foo": "foo", "foo1": "foo1"})
+    run_copy("foo", "bar", name="copy-foo-bar")
+    run_copy("bar", "foobar", name="copy-bar-foobar")
+
+    run_copy("foo1", "bar1", name="copy-foo-bar", force=True)
+
+    data = parse_yaml_for_update(
+        (tmp_dir / PIPELINE_FILE).read_text(), PIPELINE_FILE
+    )
+    assert list(data["stages"].keys()) == ["copy-foo-bar", "copy-bar-foobar"]
+
+
+def test_run_overwrite_preserves_meta_and_comment(tmp_dir, dvc, run_copy):
```

Comment:
Is this a feature? I am not sure. Preserving this took less effort

Reply:
@skshetry comments are indeed great to preserve, but meta indeed no longer seems very relevant. But at the same time I'm not sure if anyone is using it. But since we support it already - let's keep it for now.